src/plugins/intel_cpu/tests/unit/dnnl_memory_test.cpp

```diff
@@ -99,6 +99,6 @@ TEST(StaticMemoryTest, UnsupportedDnnlPrecision) {
     OV_ASSERT_NO_THROW(raw_data_ptr = testMemory->getData());
     ASSERT_FALSE(nullptr == raw_data_ptr);
     dnnl_memory = dnnl::memory();
-    ASSERT_THROW(dnnl_memory = testMemory->getPrimitive(), ov::Exception);
+    OV_ASSERT_NO_THROW(dnnl_memory = testMemory->getPrimitive());
```

Comment:
I guess we discussed this in a separate PR where you introduced a correct empty dnnl::memory::desc handling. So please revert this change.